techniques/system/inventory/1.0/fusionAgent.st

```diff
@@ -60,9 +60,8 @@ bundle agent doInventory
   methods:
     ignore_disable_inventory|!disable_inventory::
       "any" usebundle => doInventory_always;
-  reports:
     !(ignore_disable_inventory|!disable_inventory)::
-      "INFO: The file ${g.rudder_disable_inventory_file} is present. Rudder will not send an inventory. Use rudder agent inventory --force to override this temporarily";
+      "any" usebundle => rudder_common_report("Inventory", "log_info", "&TRACKINGKEY&", "inventory", "None", "The file ${g.rudder_disable_inventory_file} is present, so Rudder will not send an inventory. Use 'rudder agent inventory -f' to override this temporarily");
```

Comment:
Hmmm, this looks like the 2.11 version says "--force". Could you make a separate PR to correct that to "-f"?